src/stores/__mocks__/plan.mock.ts

```diff
@@ -0,0 +1,69 @@
+import { derived, writable, type Readable, type Writable } from 'svelte/store';
```

Comment:
Is there a way to use the original `plan` store without needing to duplicate everything? I'm worried about keeping the two in sync.

Reply:
There might be some clever ways. I haven't explored too many options yet. I just went with what I found here https://github.com/sveltejs/kit/discussions/9759